package.json

```diff
@@ -19,10 +19,38 @@
     "config"
   ],
   "scripts": {
+    "check-branch": "enkidevs-assert-restricted-branch",
+    "format": "prettier --write 'packages/**/*.js'",
+    "lint": "eslint . --ignore-path .gitignore",
+    "lint:ci": "CI=true eslint . --ignore-path .gitignore --quiet",
```

Comment:
[FYI: Circle will automatically set `CI=true` ](https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables)